superdesk.config.js

```diff
@@ -8,6 +8,15 @@
 module.exports = function(grunt) {
     return {
         defaultRoute: '/workspace',
-        requiredMediaMetadata: ['headline', 'description_text', 'alt_text']
+        requiredMediaMetadata: ['headline', 'description_text', 'alt_text'],
+        publisher: {
```

Comment:
It might make more sense to remove the `features.webPublisher` configuration key completely, and instead simply check that this key (`publisher`) is defined. Otherwise it seems a bit duplicated. I mean, if people define this object, then they obviously want to use web publisher. What do you think?